ansible/roles/sap-hana-scaleout-standby/tasks/master2.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Deploy gceStorageClient global.ini
+  template:
```

Comment:
The `template` module already assumes `{{ role_path }}/templates`, so only `global_ini.j2` is needed.